src/shared/components/palette/text-area/write-text-area-field.html

```diff
@@ -1,11 +1,12 @@
 <div class="form-group" [ngClass]="{'form-group-error': !textareaControl.valid && textareaControl.touched}">
 
   <label [for]="id()">
-    <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
-    <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
-    <span class="error-message" *ngIf="textareaControl.errors && textareaControl.touched">{{textareaControl.errors | ccdFirstError}}</span>
+    <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>    
   </label>
+  <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
+  <span class="error-message" *ngIf="textareaControl.errors && textareaControl.touched">{{textareaControl.errors | ccdFirstError:caseField.label}}</span>
 
-  <textarea class="form-control bottom-30" [id]="id()" rows="3" [formControl]="textareaControl"></textarea>
+  <textarea class="form-control bottom-30" [ngClass]="{'govuk-textarea--error': textareaControl.errors && textareaControl.touched}" 
+  [ngStyle]="{'border-width':textareaControl.errors && textareaControl.touched ? '2px':null}" [id]="id()" rows="3" [formControl]="textareaControl"></textarea>
```

Comment:
Same as above

Reply:
As discussed removed the [ngStyle] attribute